tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -71,26 +71,33 @@ cuda_py_test(
     ],
 )
 
+py_library(
+    name = "benchmark_util",
+    srcs = ["benchmark_util.py"],
+    deps = [
+        "//tensorflow:tensorflow_py",
+        "//third_party/py/numpy",
+    ],
+)
+
 py_test(
-    name = "keras_examples_benchmark_test",
+    name = "bidirectional_lstm_benchmark_test",
     size = "medium",
-    srcs = ["keras_examples_benchmark_test.py"],
+    srcs = ["keras_examples_benchmarks/bidirectional_lstm_benchmark_test.py"],
```

Comment:
Let's add some tags for now to avoid breaks: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/benchmarks/BUILD#L80-L81